java/src/jmri/jmrit/symbolicprog/VariableValue.java

```diff
@@ -408,12 +434,19 @@ protected int maskVal(String maskString) {
                     mask = mask | 1;
                 }
             } catch (StringIndexOutOfBoundsException e) {
-                log.error("mask /" + maskString + "/ could not be handled for variable " + label());
+                log.error("mask \"{}\" could not be handled for variable ", label());
```

Comment:
Wouldn't you want to include the variable ``maskString`` to fill in the first {} like: 
``mask \"{}\" could not be handled for variable {}", maskString, label()``

Reply:
Good catch, will fix.  Thanks!